scripts/operation.py

```diff
@@ -104,38 +99,50 @@ def get_signer() -> Any:
         help='Transactions file path. Each transaction per line',
         required=True,
     )
+    current_parser.add_argument(
+        "--phase",
+        type=str,
+        help='Phase of deploy: preparation, deactivation, upgrade, adding',
+        required=True,
+    )
 
     current_parser = subparsers.add_parser(
-        'check-transactions',
-        help='Check transactions from a file',
+        'check-transactions', help='Check transactions from a file'
     )
     current_parser.add_argument(
-        "--transactions-path",
+        "--phase",
         type=str,
-        help='Path to transactions file',
+        help='Phase of deploy: preparation, deactivation, upgrade, adding',
         required=True,
     )
+    current_parser.add_argument(
+        "--transactions-path", type=str, help='Path to transactions file', required=True
+    )
+
+    current_parser = subparsers.add_parser('test', help='`Command for tests`')
 
     args = parser.parse_args()
 
     sys.argv.append('--verbose')
 
-    with open(args.config) as f:
+    install_solido()
+    with open(os.getenv("SOLIDO_CONFIG")) as f:
         config = json.load(f)
         cluster = config.get("cluster")
         if cluster:
             os.environ['NETWORK'] = cluster
 
     if args.command == "deactivate-validators":
-        lido_state = solido('--config', args.config, 'show-solido')
+        set_solido_cli_path(os.getenv("SOLIDO_V1"))
```

Comment:
This is not a good idea. We should not use hard coded environment variables. Instead we should pass them as CLI arguments

Reply:
The option `--config` has been hidden to reduce the load on multisig owners and simplify instructions for them.
A good interface design framework here is simplification to reduce the number of errors in use.